From a3486be42b85b79e7faa4b9f89c720f6e14d722a Mon Sep 17 00:00:00 2001
From: Jo Shields <jo.shields@xamarin.com>
Date: Thu, 27 Aug 2015 12:39:48 +0100
Subject: [PATCH] Make "NuGet.exe restore" conditional on git.

As per 6e7f2dfcbc80271fdebdee5418b5ed80fe046086, "nuget restore" should not be called in xbuilds in Tarball builds (due to missing NuGet.exe bundle)
---
 main/tests/UserInterfaceTests/UserInterfaceTests.csproj | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/main/tests/UserInterfaceTests/UserInterfaceTests.csproj b/main/tests/UserInterfaceTests/UserInterfaceTests.csproj
index 3f865b5..d175bbb 100644
--- a/main/tests/UserInterfaceTests/UserInterfaceTests.csproj
+++ b/main/tests/UserInterfaceTests/UserInterfaceTests.csproj
@@ -130,7 +130,7 @@
       <NuGet>$(SolutionDir)\external\nuget-binary\NuGet.exe</NuGet>
       <NuGet Condition="$(OS)=='Unix'">mono $(NuGet)</NuGet>
     </PropertyGroup>
-    <Exec Command="$(NuGet) restore -SolutionDirectory $(SolutionDir)" />
+    <Exec Condition="Exists('$(SolutionDir)\..\.git')" Command="$(NuGet) restore -SolutionDirectory $(SolutionDir)" />
   </Target>
   <ItemGroup>
     <Folder Include="DialogTests\" />
@@ -138,4 +138,4 @@
     <Folder Include="Exceptions\" />
     <Folder Include="VersionControlTests\Git\" />
   </ItemGroup>
-</Project>
\ No newline at end of file
+</Project>
-- 
1.9.1

